<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="159834" started-at="2012-08-01T06:42:00Z" finished-at="2012-08-01T06:44:40Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="159834" started-at="2012-08-01T06:42:00Z" finished-at="2012-08-01T06:44:40Z">
      <class name="com.registration.tests.Step2">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.registration.tests.Step2@5488e2dd]" name="setUp" is-config="true" duration-ms="3556" started-at="2012-08-01T10:42:00Z" finished-at="2012-08-01T10:42:04Z">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testStep2(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.registration.tests.Step2@5488e2dd]" name="testStep2" duration-ms="76501" started-at="2012-08-01T10:42:04Z" data-provider="step2" finished-at="2012-08-01T10:43:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Тестов]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Тест]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Тестович]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[test@test.ru]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[//div[@id='edit-company-scope-wrapper']/span/ul/li[2]]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[timeout after click on button 'Next']]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: timeout after click on button 'Next'
	at org.testng.Assert.fail(Assert.java:89)
	at com.registration.tests.Step2.testStep2(Step2.java:118)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:715)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:907)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1237)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testStep2 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.registration.tests.Step2@5488e2dd]" name="tearDown" is-config="true" duration-ms="92" started-at="2012-08-01T10:43:20Z" finished-at="2012-08-01T10:43:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.registration.tests.Step2@5488e2dd]" name="setUp" is-config="true" duration-ms="3918" started-at="2012-08-01T10:43:21Z" finished-at="2012-08-01T10:43:24Z">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testStep2(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.registration.tests.Step2@5488e2dd]" name="testStep2" duration-ms="75672" started-at="2012-08-01T10:43:24Z" data-provider="step2" finished-at="2012-08-01T10:44:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[123]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[456]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[789]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[111@111.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[//div[@id='edit-company-scope-wrapper']/span/ul/li[3]]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[-e]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[timeout after click on button 'Next']]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: timeout after click on button 'Next'
	at org.testng.Assert.fail(Assert.java:89)
	at com.registration.tests.Step2.testStep2(Step2.java:118)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:715)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:907)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1237)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testStep2 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.registration.tests.Step2@5488e2dd]" name="tearDown" is-config="true" duration-ms="74" started-at="2012-08-01T10:44:40Z" finished-at="2012-08-01T10:44:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.registration.tests.Step2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
